src/static/js/app/components/Survey.js

```diff
@@ -0,0 +1,92 @@
+import 'fetch';
```

Comment:
I ended up breaking the whole Survey component into it's own file simply for testing purposes... *sigh*, and then mocking `fetch` happened.